pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -123,59 +129,73 @@ func NewStandardSecurityPolicyEnforcer(containers []securityPolicyContainer, enc
 	}
 }
 
-func (c Containers) toInternal() ([]securityPolicyContainer, error) {
+func (c *Containers) toInternal() ([]securityPolicyContainer, error) {
```

Comment:
why the change of all the `toInternal` to be on the pointer rather than the object itself?